Created v1.5.2 relase notes (#5373)
dotnet/machinelearning: docs/release-notes/1.5.2/release-1.5.2.md

- Fixed OnnxTransformer output column mapping. ([#5192](https://github.com/dotnet/machinelearning/pull/5192))
- Fixed version format of built packages ([#5197](https://github.com/dotnet/machinelearning/pull/5197))
- Improvements to "Invalid TValue" error message ([#5189](https://github.com/dotnet/machinelearning/pull/5189))
- Added IDisposable to OnnxTransformer and fixed memory leaks ([#5348](https://github.com/dotnet/machinelearning/pull/5348))

> - Fixes #4392 Add AddPredictionEnginePool overload for implementation factory ([#4393](https://github.com/dotnet/machinelearning/pull/4393))

Reviewer: Recommend full URLs as otherwise the auto-numeric links break in github if a repo is renamed, or if viewing within a fork.
```suggestion
- Fixes [#4392](https://github.com/dotnet/machinelearning/issues/4392) Add AddPredictionEnginePool overload for implementation factory ([#4393](https://github.com/dotnet/machinelearning/pull/4393))
```

- Updated codegen to make it work with mlnet 1.5  ([#5173](https://github.com/dotnet/machinelearning/pull/5173))
- Updated codegen to support object detection scenario. ([#5216](https://github.com/dotnet/machinelearning/pull/5216))

> - Fix issue #5350, check file lock before reload model ([#5351](https://github.com/dotnet/machinelearning/pull/5351))

Reviewer: Recommend full URLs as otherwise the auto-numeric links break in github if a repo is renamed, or if viewing within a fork.
```suggestion
- Fix issue [#5350](https://github.com/dotnet/machinelearning/issues/5350), check file lock before reload model ([#5351](https://github.com/dotnet/machinelearning/pull/5351))
```

> - AutoML.Net filter infinity value when calculate average score ([#5345](https://github.com/dotnet/machinelearning/pull/5345))

Reviewer: ```suggestion
- Improve handling of infinity values in AutoML.NET when calculating average CV metrics ([#5345](https://github.com/dotnet/machinelearning/pull/5345))
```

- Throw when PCA generates invalid eigenvectors ([#5349](https://github.com/dotnet/machinelearning/pull/5349))
- RobustScalingNormalizer entrypoint added ([#5310](https://github.com/dotnet/machinelearning/pull/5310))
- Replace whitelist terminology to allow list ([#5328](https://github.com/dotnet/machinelearning/pull/5328)) (Thanks [**@LetticiaNicoli**](https://github.com/LetticiaNicoli))

> - Fixes #5352 issues caused by equality with non-string values for root cause localization  ([#5354](https://github.com/dotnet/machinelearning/pull/5354))

Reviewer: Recommend full URLs as otherwise the auto-numeric links break in github if a repo is renamed, or if viewing within a fork.
```suggestion
- Fixes [#5352](https://github.com/dotnet/machinelearning/issues/5352) issues caused by equality with non-string values for root cause localization  ([#5354](https://github.com/dotnet/machinelearning/pull/5354))
```

> - Fixes #5352 issues caused by equality with non-string values for root cause localization  ([#5354](https://github.com/dotnet/machinelearning/pull/5354))

Reviewer 2: Nit:
```suggestion
- Fixes issue caused by equality with non-string values for root cause localization  ([#5354](https://github.com/dotnet/machinelearning/pull/5354))
```

> - Aded catch in R^2 calculation for case with few samples ([#5319](https://github.com/dotnet/machinelearning/pull/5319))

Reviewer: ```suggestion
- Added check in R^2 calculation for the case with few samples ([#5319](https://github.com/dotnet/machinelearning/pull/5319))
```

> 

Reviewer 2: Now that PR #5386 is merged, it can be added here as well, as "- Added support for RankingMetrics with CrossValSummaryRunner".
